sdcm/ec2_client.py

```diff
@@ -133,6 +134,12 @@ def _request_spot_fleet(self, instance_type, image_id, region_name, network_if,
         LOGGER.debug('Spot fleet request: %s', request_id)
         return request_id
 
+    def _get_cloud_init_yaml(self):
+        path = Path("./sdcm/provision/aws-cloud-init.txt")
+
+        with path.open(mode="r", encoding='utf-8') as stream:
```

Comment:
if you are using Path, you can use `path.read_text()`

https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text

Reply:
Good idea - fixed!